src/containers/CommunityProjects/CommunityProjectsStyles.css

```diff
@@ -0,0 +1,42 @@
+.cardFlexBox {
```

Comment:
Not familiar with how CRA handles these styles. Are these styled scoped in any way? 

It would be great to introduce styled components into this project but this is out of scope of this PR.

Reply:
Yeah I also thought about using styled-components but didn't want to introduce any big changes. I didn't see any styling conflicts in the build as of right now, so the scope issue sorta went over my head.

One way to ensure there's no styling conflicts could be to just give the parent div an id and use it as a specific selector for all the appropriate styles (but this seems less than ideal)

Reply:
Take a look at CSS modules.

https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

It looks like it may be built into create react app. This would scope the CSS to prevent global clashing.

Reply:
Alright so I used the module.css and it worked pretty well. I still had to wrap the entire page in a communityProjectPage class. Since Polaris doesn't let you pass custom styles, this is the only way I could see to target Polaris styles without the css changes clashing with the Global,

55a4d9d